util/ranger/ranger_test.go

```diff
@@ -1443,6 +1443,7 @@ func (s *testRangerSuite) TestIndexRangeForBit(c *C) {
 	c.Assert(err, IsNil)
 	testKit := testkit.NewTestKit(c, store)
 	testKit.MustExec("use test;")
+	testKit.MustExec("set @@tidb_partition_prune_mode = 'static-only';")
```

Comment:
It seems that the value of the variable `tidb_partition_prune_mode` in the original test defaulted to `dynamic-only`. It's unexpected.

Reply:
In this [PR](https://github.com/pingcap/tidb/pull/19911), we enable `Dynamic Prune` by default in the test case. You can see [here](https://github.com/lysu/tidb/blob/c63cc487a17ae463c79aaca4145065e7665c693b/session/bootstrap.go#L1267-L1273) for more details.